[2024-01-28T13:42:40.243+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data scheduled__2023-01-17T00:00:00+00:00 [queued]>
[2024-01-28T13:42:40.381+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data scheduled__2023-01-17T00:00:00+00:00 [queued]>
[2024-01-28T13:42:40.403+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 6
[2024-01-28T13:42:40.949+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): get_stackoverflow_data> on 2023-01-17 00:00:00+00:00
[2024-01-28T13:42:40.985+0000] {standard_task_runner.py:60} INFO - Started process 988 to run task
[2024-01-28T13:42:41.039+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'fetch_stackoverflow_data', 'get_stackoverflow_data', 'scheduled__2023-01-17T00:00:00+00:00', '--job-id', '999', '--raw', '--subdir', 'DAGS_FOLDER/loading_data_to_aws_S3.py', '--cfg-path', '/tmp/tmp4bi21dss']
[2024-01-28T13:42:41.070+0000] {standard_task_runner.py:88} INFO - Job 999: Subtask get_stackoverflow_data
[2024-01-28T13:42:41.438+0000] {task_command.py:423} INFO - Running <TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data scheduled__2023-01-17T00:00:00+00:00 [running]> on host 9924936b75b5
[2024-01-28T13:42:42.007+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='charlotte' AIRFLOW_CTX_DAG_ID='fetch_stackoverflow_data' AIRFLOW_CTX_TASK_ID='get_stackoverflow_data' AIRFLOW_CTX_EXECUTION_DATE='2023-01-17T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-01-17T00:00:00+00:00'
[2024-01-28T13:42:42.916+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-01-28T13:42:43.026+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=fetch_stackoverflow_data, task_id=get_stackoverflow_data, execution_date=20230117T000000, start_date=20240128T134240, end_date=20240128T134243
[2024-01-28T13:42:44.013+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-28T13:42:46.107+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
